data/json/items/armor/suits_protection.json

```diff
@@ -1196,23 +1196,57 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "scrap suit" },
-    "description": "A suit of armor made from scraps of metal secured by simple strings; the loose collection of plates provides decent but not the most convenient protection.",
+    "description": "A suit of armor made from scraps of metal secured by simple strings; the loose collection of plates provides decent but uncomfortable protection.",
     "weight": "9534 g",
     "volume": "15500 ml",
     "price": 60000,
     "price_postapoc": 250,
     "to_hit": -5,
     "bashing": 6,
     "cutting": 8,
-    "material": [ "steel", "iron" ],
     "symbol": "[",
     "looks_like": "armor_lightplate",
     "color": "light_gray",
-    "warmth": 10,
     "longest_side": "40 cm",
-    "material_thickness": 2,
     "flags": [ "OUTER" ],
-    "armor": [ { "encumbrance": 18, "coverage": 80, "covers": [ "torso", "leg_l", "leg_r", "arm_l", "arm_r" ] } ]
+    "armor": [
+      {
+        "material": [
+          { "type": "budget_steel", "covered_by_mat": 75, "thickness": 2 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 1.0 }
+        ],
+        "covers": [ "torso" ],
+        "coverage": 100,
+        "cover_melee": 80,
```

Comment:
if cover melee and cover ranged are 80 then all attacks will be at 80 coverage so why does this have 100% coverage?

Also same suggestions that the armor should have at least 1 material have 100% coverage and the armor coverage should be scaled down to compensate.